core/src/language/python.rs

```diff
@@ -275,27 +275,31 @@ impl Language for Python {
             // Write all the unit variants out (there can only be unit variants in
             // this case)
             RustEnum::Unit(shared) => {
-                self.add_import("typing".to_string(), "Literal".to_string());
-                write!(
-                    w,
-                    "{} = Literal[{}]",
-                    shared.id.renamed,
+                self.add_import("enum".to_string(), "Enum".to_string());
```

Comment:
We can just have this at the start of the function because no matter what we need to import this. For unit or algebraic enums.